tbx/project_styleguide/templates/patterns/organisms/footer/footer.html

```diff
@@ -1,53 +1,58 @@
+        {% include "patterns/molecules/footer-cta/footer-cta.html"%}
 
-    {% footerlinks %}
+        <div class="grid__footer-links">
+            {% footerlinks %}
+            <p class="footer__copyright">© Torchbox {% now "Y" %}</p>
 
-    <ul>
-        <li>
-            <a aria-label="Follow us on Twitter" href="https://twitter.com/torchbox">
-                <svg width="24" height="24" aria-hidden="true">
-                    <use xlink:href="#twitter" />
-                </svg>
-            </a>
-        </li>
-        <li>
-            <a aria-label="Connect with us on LinkedIn" href="https://www.linkedin.com/company/torchbox">
-                <svg width="24" height="24" aria-hidden="true">
-                    <use xlink:href="#linkedin" />
-                </svg>
-            </a>
-        </li>
-        <li>
-            <a aria-label="Follow us on Instagram" href="https://www.instagram.com/torchboxltd/">
-                <svg width="24" height="24" aria-hidden="true">
-                    <use xlink:href="#instagram" />
-                </svg>
-            </a>
-        </li>
-    </ul>
+            <ul class="footer__socials">
+                <li>
+                    <a aria-label="Follow us on Twitter" href="https://twitter.com/torchbox" class="footer__social-link">
+                        <svg width="24" height="24" aria-hidden="true" class="footer__social-icon">
+                            <use xlink:href="#twitter" />
+                        </svg>
+                    </a>
+                </li>
+                <li>
+                    <a aria-label="Connect with us on LinkedIn" href="https://www.linkedin.com/company/torchbox" class="footer__social-link">
+                        <svg width="24" height="24" aria-hidden="true" class="footer__social-icon">
+                            <use xlink:href="#linkedin" />
+                        </svg>
+                    </a>
+                </li>
+                <li>
+                    <a aria-label="Follow us on Instagram" href="https://www.instagram.com/torchboxltd/" class="footer__social-link">
+                        <svg width="24" height="24" aria-hidden="true" class="footer__social-icon">
+                            <use xlink:href="#instagram" />
+                        </svg>
+                    </a>
+                </li>
+            </ul>
+        </div>
+    </div>
+
+    <div class="grid footer__bottom">
+        <div class="grid__footer-logos">
+            {% with global_settings=settings.navigation.NavigationSettings %}
+                {% if global_settings.footer_logos %}
+                    <ul class="footer__logos">
+                        {% for item in global_settings.footer_logos %}
+                            {% with footer_logo=item.value.image %}
+                                <li>
+                                    {% image footer_logo max-100x100 format-webp loading="lazy" class="footer__logo" %}
```

Comment:
There's an option in Wagtail to add a link to these - is this being removed or do we need to account for that in the FE?

![image](https://github.com/torchbox/torchbox.com/assets/31627284/1c21a388-fef3-4ebe-9cfb-3e68fc463149)

Reply:
That link option is being removed in the back-end as at the moment it is used inconsistently.